src/org/labkey/targetedms/query/ConflictResultsManager.java

```diff
@@ -88,67 +87,76 @@ private ConflictResultsManager() {}
         getConflictProteinsSql.add(RepresentativeDataState.Conflicted.ordinal());
         getConflictProteinsSql.add(RepresentativeDataState.Representative.ordinal());
 
-        ConflictProtein[] pepGroups = new SqlSelector(TargetedMSManager.getSchema(), getConflictProteinsSql).getArray(ConflictProtein.class);
-        return Arrays.asList(pepGroups);
+        return new SqlSelector(TargetedMSManager.getSchema(), getConflictProteinsSql).getArrayList(ConflictProtein.class);
     }
 
     public static List<ConflictPrecursor> getConflictedPrecursors(Container container)
+    {
+        List<ConflictPrecursor> result = new ArrayList<>();
+        result.addAll(getConflictedPrecursors(container, TargetedMSManager.getTableInfoPeptide(), TargetedMSManager.getTableInfoPrecursor(), "ModifiedSequence", new SQLFragment("prec.ModifiedSequence"),true));
+        result.addAll(getConflictedPrecursors(container, TargetedMSManager.getTableInfoMolecule(), TargetedMSManager.getTableInfoMoleculePrecursor(), "IonFormula", new SQLFragment("COALESCE(prec.CustomIonName, prec.IonFormula)"), false));
```

Comment:
Prefer prec.IonFormula over prec.CustomIonName in the COALESCE since precursors of a molecule will have the same CustomIonName.  Both IonFormula and CustomIonName are nullable, so maybe add GeneralPrecursor.mz here as it is done in molExprCol in MoleculeTableInfo?

Reply:
Changed this to concatenate the values, plus mz